tensorboard/uploader/formatters_test.py

```diff
@@ -97,7 +115,7 @@ def testJsonFormatterWithEmptyNameAndDescription(self):
         util.set_timestamp(experiment.update_time, 1015218367)
         experiment_url = "http://tensorboard.dev/deadbeef"
         formatter = formatters.JsonFormatter()
-        output = formatter.format_experiment(experiment, experiment_url)
+        output = self._format(formatter, experiment, experiment_url)
```

Comment:
Can we test that the JSON formatter *is* timezone-independent even
though the readable formatter is not?

Reply:
Done.